### DISCUSSION TITLE
Custom Fields for Version 2.0

### DISCUSSION THREAD

**Discussion by anunezITS** [kimai/kimai#3961] [Plugins]
Hello, We are interested in buying the custom fields for Kimai version 2.0. https://www.kimai.org/store/custom-fields-bundle.html
is the bundle compatible already?  If not can you please provide a timeline. 

Thanks,
Albert

**Comment by kevinpapst** [comment]
Kimai 2.0, which is the demo version you reach at https://www.kimai.org/en/demo.html via the yellow button, is not yet officially released. Probably end of the month.
